bug_actiongroup_page.php

```diff
@@ -294,7 +294,7 @@
 			switch( $f_action ) {
 				case 'COPY':
 				case 'MOVE':
-					print_project_option_list( null, false );
+					print_project_option_list( null, false, null, false, true /* $p_can_report_only */ );
```

Comment:
Please add the parameter names next to each parameter for readability.

Reply:
ok, done